.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:
   # https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds
   - flake8 . --count --exit-zero --statistics --select=E,F,W,C90
 script:
-  - coverage run -m pytest
+  - coverage run -m pytest test
```

Comment:
Could you add pytest integration after the coverage checking? We want to make sure that integration tests pass before merging (but as a separate stage)

Reply:
this includes the integration tests since the `integration/` folder is a subdirectory

Reply:
I was saying to run them seperately. The idea being that if unit tests fail, we don't want to bother running the integration tests since they're slow

Reply:
gotcha, will do!